src/lib/libtime.js

```diff
@@ -362,21 +362,21 @@ addToLibrary({
 
       // seconds
       if ((value=getMatch('S'))) {
-        date.sec = jstoi_q(value);
+        date.sec = Number(value);
```

Comment:
The code above is fairly complex - how can I see that this is a pure number without a prefix?

Reply:
The `getMatch` function returns that result of the regex's above.

You can see that we only call `Number` on the following matches: S, M, H, I, Y, y, C, m, d, j, U, W.

You can look at the above table of regexes and see that each of them only ever match numbers.

We also have pretty extensive testing for strftime I believe.